### PR TITLE
Extension Manifest Converter blog post

### PR DESCRIPTION
**Pull request by dotproto**
Revised extension manifest converter blog post. 

https://deploy-preview-708--developer-chrome-com.netlify.app/blog/extension-manifest-converter/

### CODE REVIEW

File: site/en/blog/extension-manifest-converter/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Extension Manifest Converter
```
Comment (awfuchs) [inline site/en/blog/extension-manifest-converter/index.md:3]: This sounds like it only converts the manifest. Would "Manifest V3 extension converter" make more sense?

Also we use sentence case for titles, e.g. "Extension manifest converter" if we were keeping the title unchanged.

File: site/en/blog/extension-manifest-converter/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  - extensions
+hero: image/WlD8wC6g8khYWPJUsQceQkhXSlv1/GVDuhpIli5DeYmbiPlqG.jpg
+alt: A woodworking hand plane.
+---
+
+Hi everyone. My name is Solomon and I'm a software engineer on Chrome's extensions team.
+
+As we continue to build out the latest version of [Chrome's extensions platform][mv3], I found
+myself needing to convert MV2 extensions to MV3 for testing purposes. To make this process a little
+easier I created a tool called [Extension Manifest Converter][emc-repo] (EMC).
```
Comment (awfuchs) [inline site/en/blog/extension-manifest-converter/index.md:24]: Ah yes, I see this is actually what we named it. Is that set in soapstone?

File: site/en/blog/extension-manifest-converter/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+```bash
+python3 extension.py <extension_path>
+```
+
+This tool automates some of the more mechanical parts of updating an extension like moving from
+`chrome.browserAction` to `chrome.action` or moving to the new [web accessible resources][war]
+syntax introduced with Manifest V3.
+
+If you encounter any issues with the project, [open an issue][emc-issue] on the [project's
```
Comment (awfuchs) [inline site/en/blog/extension-manifest-converter/index.md:38]: This is the rare case where "please" is appropriate: "...please open an issue..."

https://developers.google.com/style/word-list?hl=en#:~:text=Use%20please%20only,your%20account%20representative.

**Review by awfuchs — APPROVED** [review]

File: site/en/blog/extension-manifest-converter/index.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+alt: A woodworking hand plane.
+---
+
+Hi everyone. My name is Solomon and I'm a software engineer on Chrome's extensions team.
+
+As we continue to build out the latest version of [Chrome's extensions platform][mv3], I found
+myself needing to convert MV2 extensions to MV3 for testing purposes. To make this process a little
+easier I created a tool called [Extension Manifest Converter][emc-repo] (EMC).
+
+EMC is a Python 3 command line tool that automates several parts of converting an extension form MV2
```
Comment (rdcronin) [inline site/en/blog/extension-manifest-converter/index.md:26]:
```suggestion
EMC is a Python 3 command line tool that automates several parts of converting an extension from MV2
```

File: site/en/blog/extension-manifest-converter/index.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+tags:
+  - extensions
+hero: image/WlD8wC6g8khYWPJUsQceQkhXSlv1/GVDuhpIli5DeYmbiPlqG.jpg
+alt: A woodworking hand plane.
+---
+
+Hi everyone. My name is Solomon and I'm a software engineer on Chrome's extensions team.
+
+As we continue to build out the latest version of [Chrome's extensions platform][mv3], I found
+myself needing to convert MV2 extensions to MV3 for testing purposes. To make this process a little
```
Comment (rdcronin) [inline site/en/blog/extension-manifest-converter/index.md:23]: MV2 + MV3 seem largely like they should be internal jargon, though they do seem to have leaked a bit.  I think we generally still use "Manifest V3" though.  Andy?

File: site/en/blog/extension-manifest-converter/index.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+to MV3. Users can quickly convert an extension directory, zip file, or manifest.json file with a
+single command.
+
+```bash
+python3 extension.py <extension_path>
+```
+
+This tool automates some of the more mechanical parts of updating an extension like moving from
+`chrome.browserAction` to `chrome.action` or moving to the new [web accessible resources][war]
+syntax introduced with Manifest V3.
```
Comment (rdcronin) [inline site/en/blog/extension-manifest-converter/index.md:36]: While we link to the repo (which includes this), I wonder if we should explicitly call out:
a) Look at the repo's .md for a more thorough description of what this handles
b) This does not automate everything - in particular, runtime and behavior changes like adopting service worker are not supported
c) This is a tool we found useful, and wanted to open source.  It's not going to have first-party, dedicated, perennial support.  YMMV.  (In fewer words : ))

File: site/en/blog/extension-manifest-converter/index.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+subhead: >
+  Easily convert an entire directory, extension zip file, or manifest.json file.
+date: 2021-04-28
+updated: 2021-04-28
+authors:
+  - solomonkinard
+  - dotproto
+tags:
+  - extensions
+hero: image/WlD8wC6g8khYWPJUsQceQkhXSlv1/GVDuhpIli5DeYmbiPlqG.jpg
```
Comment (rdcronin) [inline site/en/blog/extension-manifest-converter/index.md:16]: While a good picture - what's the correlation between planing and MV2->3 conversion? ; )

**Review by rdcronin — COMMENTED** [review]
Thanks, Simeon!

File: site/en/blog/extension-manifest-converter/index.md (hunk 7 above)
Comment (dotproto) [reply site/en/blog/extension-manifest-converter/index.md:16]: It's a pretty loose association. The vague idea is that like a plane this is specialized tool for a specific job. It won't do everything for you, but it is useful for the niche it fills. Nothing beyond that besides its visual appeal.

I'm open to other ideas. I also have a couple other contenders that might be worth discussing offline.

File: site/en/blog/extension-manifest-converter/index.md (hunk 5 above)
Comment (dotproto) [reply site/en/blog/extension-manifest-converter/index.md:23]: That's on me. I get lazy and stop typing out "Manifest V3." I'm open to updating this post & avoiding the MVX pattern moving forward.

File: site/en/blog/extension-manifest-converter/index.md (hunk 3 above)
Comment (dotproto) [reply site/en/blog/extension-manifest-converter/index.md:38]: Haha, I wrote and deleted please more times than I'd like to admit. I will add it back.

File: site/en/blog/extension-manifest-converter/index.md (hunk 2 above)
Comment (dotproto) [reply site/en/blog/extension-manifest-converter/index.md:24]: I have to defer to @solomonkinard on that

File: site/en/blog/extension-manifest-converter/index.md (hunk 1 above)
Comment (dotproto) [reply site/en/blog/extension-manifest-converter/index.md:3]: Updated the title to use sentence case.

File: site/en/blog/extension-manifest-converter/index.md (hunk 6 above)
Comment (dotproto) [reply site/en/blog/extension-manifest-converter/index.md:36]: These are all good suggestions. Working on integrating them now.

File: site/en/blog/extension-manifest-converter/index.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Extension manifest converter
+description: >
+  Open source tool to convert extensions to Manifest V3. You'll still need to manually update any
+  service worker code that relies on a DOM, in addition to updating chrome.scripts.
```
Comment (rdcronin) [inline site/en/blog/extension-manifest-converter/index.md:6]:
```suggestion
  Open source tool to convert extensions to Manifest V3. You'll still need to manually update any
  code with non-mechanical changes, such as adapting to use service workers or some script
  injection.
```

(Trying to make it more obvious that this is a non-exhaustive list)

**Review by rdcronin — APPROVED** [review]
LGTM % nit.  Thanks, Simeon!

**Pull request merged**